.github/workflows/ci.yaml

```diff
@@ -0,0 +1,125 @@
+name: CI
+
+on: [push]
+
+jobs:
+  cvc4:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Inspect
+      run: |
+         echo $PWD
+         find .
+    - name: Get CVC4
+      run: |
+        git clone -b eqrange-quant --single-branch https://github.com/GaloisInc/vadd-CVC4.git .
+    - name: Get Antlr
+      run: ./contrib/get-antlr-3.4
+    - name: Configure
+      run: ./configure.sh production --static --proofs --prefix=$PWD/dist
+    - name: Build
+      working-directory: build
+      run: make -j 8
+    - name: Package
+      run: |
+        find build -name cvc4
+        readelf -d build/bin/cvc4
+    - name: cvc4 artifact
+      uses: actions/upload-artifact@v1
+      with:
+        name: cvc4
+        path: build/bin/cvc4
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # NOTE: must list all targets in `include` here to activate them
+        name: ["Linux 8.8.3"]
+        include:
+          - name: Linux 8.8.3
+            ghc-ver: 8.8.3
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Get GHC
+      run: |
+        sudo add-apt-repository ppa:hvr/ghc
+        sudo apt-get update
```

Comment:
The ubuntu environments already come with the hvr/ghc repository. In fact, `actions/setup-haskell` uses it as an implementation detail.

```yaml
- name: Get GHC
  uses: actions/setup-haskell
  with:
    ghc-version: ${{ matrix.ghc-ver }}
    cabal-version: '3.0'
```

should be sufficient, although I have an [open PR](https://github.com/actions/setup-haskell/pull/12) that rewrites the entire action to be much more flexible and useful. You can use it at `haskell-CI/setup@master` now if you really want to.

Reply:
What's the advantage of using this action?  Is it faster somehow?

Reply:
The action will be faster if the runner comes with that version of GHC (it just does a path check in that case). It'll also soon work on windows, macOS, and support arbitrary GHC versions. More importantly, the actions ensure PATHs and other environment setups are done correctly so you don't have to think about it.

Reply:
Conversely actions require me to understand what the action is doing -- it's pretty easy to read what a command is doing.